airflow/www/static/footable.core.css

```diff
@@ -0,0 +1,218 @@
+@font-face {
```

Comment:
We should not be copying files into this project. How many of these files are external?